src/util/chalk.js

```diff
@@ -0,0 +1,7 @@
+/* @flow */
+
+const chalk = require('chalk');
+
+chalk.enabled = process.stdout.isTTY;
```

Comment:
Kinda, this doesn't rely on the complex logic inside supports-color, but rather takes the simpler approach to just look at if it's a TTY.

Perhaps this is no good, and then I'll just remove this. Or change it to only overwrite if it _isn't_ a TTY.